java/src/jmri/jmrix/mqtt/MqttAdapter.java

```diff
@@ -287,18 +287,20 @@ public void connectionLost(Throwable thrwbl) {
     @API(status=API.Status.INTERNAL)
     public void messageArrived(String topic, MqttMessage mm) throws Exception {
         log.debug("Message received, topic : {}", topic);
-        
+
         boolean found = false;
-        for (Map.Entry<String,ArrayList<MqttEventListener>> e : mqttEventListeners.entrySet()) {
+        Map<String,ArrayList<MqttEventListener>> tempMap
+            = new HashMap<String,ArrayList<MqttEventListener>> (mqttEventListeners); // Avoid CME
```

Comment:
It would be better to write `ConcurrentModificationException` instead of `CME`.
I didn't understand the comment until I read the comment in the PR, probably since ConcurrentModificationException is an exception that happens rarely.